src/components/views/rooms/RoomSublist.tsx

```diff
@@ -553,6 +553,8 @@ export default class RoomSublist extends React.Component<IProps, IState> {
     }
 
     private renderMenu(): React.ReactElement {
+        if (this.props.tagId === DefaultTagID.Suggested) return null; // not sortable
```

Comment:
Will the button still be visible but the context menu won't work?

Reply:
The button is rendered in this function so I hope not :)

Reply:
Ah, should have clicked the expand button a few more times, sorry :sweat_smile: